backend/src/dfini_hack_team4.did

```diff
@@ -0,0 +1,59 @@
+type Amount = nat64;
+type AnswerId = nat64;
+type Solution = blob;
+type TaskId = nat64;
+type TaskPayload = blob;
+type Timestamp = nat64;
+
+type TaskType = variant {
+  translate_text;
+  edit_image;
+};
+
+type TaskStatus = variant {
+  open;
+  closed;
+};
+
+type Choice = variant {
+  yes;
+  no;
+};
+
+type Vote = record {
+  voter: principal;
+  choice: Choice;
+};
+
+type Answer = record {
+  submitter: principal;
+  submission_time: Timestamp;
+  solution: Solution;
+  votes: vec Vote;
+};
+
+type Task = record {
+  submitter: principal;
+  type: TaskType;
```

Comment:
I think only the type will not be enough. 
E.g., if the type is "translation" we also have to specify into what language the text should be transalted. 
So I think we might need another input "task" or so (so we could have a high level type and depending on the type, the task-part could just be a text, e.g., describing how to edit the picture / to what language to translate etc), while the payload is the picture / input text itself..

WDYT?